.travis.yml

```diff
@@ -41,7 +38,6 @@ python:
 
 before_install:
   - "nvm install 8; nvm use 8"
-  - "npm install -g node-sass@4.9.0 clean-css@3.4.19 uglify-js@2.7.3 requirejs@2.2.0"
```

Comment:
I believe this is still needed for the Jinja templates. It can be removed when we transition to Semantic-UI